server/services/crmDisplayService.ts

```diff
@@ -92,22 +91,23 @@ export default class CrmDisplayService {
   }
 
   private getFields<T extends CrmResponse>(fields: Array<FieldOrSubHeading>, crmResponse: T): Array<FieldOrSubHeading> {
-    return fields
-      .map(field => {
-        if (isConfigField(field)) {
-          // create display field using config & api field value
-          const displayField: DisplayField = {
-            label: field.label,
-            value: this.getApiFieldValue(crmResponse, field.apiField),
-            type: field.type,
+    if (fields) {
+      return fields
+        .map(field => {
+          if (isConfigField(field)) {
```

Comment:
This has been simplified by removing `DisplayField` type and now just populating `ConfigField.value` with the api field value.